tests/scenarios/scenarios.ts

```diff
@@ -39,7 +39,7 @@ async function lts_3_28(project: Project) {
   project.linkDevDependency('ember-data', { baseDir: __dirname, resolveName: 'ember-data' });
 }
 
-async function release(project: Project) {
+export async function release(project: Project) {
```

Comment:
Instead of this, you can say `appScenarios.only('release')`

Reply:
ooo that's handy, because I have other tests I'm working on that need just the release bit. thanks!

Reply:
Updated!